batch.m

```diff
@@ -1,20 +1,17 @@
 %% Run batches
-WD = pwd;
+opt = getOption();
+
```

Comment:
It gives an error.
Since the getOptions is a higher level in the directory level, I run the get option first, then the working directory is the subFun where all the functions are located.

Reply:
remember there was 2 getOptions function. One in the main folder and one in the subfunction.
we deleted the subfunction one, and now its only in the top folder.
So we need to run the getOptions first before defining the WD which is the subfunction folder.

Reply:
Ah I think we just did not agree on what the WD was. To me it is where the batch script is and and I add the subfun folder to the path so that all the other functions can be accessed.